capstone/client/src/components/StoresPage.js

```diff
@@ -190,21 +190,18 @@ class StorePage extends Component {
 
     const overviewCards = (this.state.originalStores.length === 0) ? <CircularProgress id="stores-loading" color="action" /> : 
     <StoreOverviewCards stores={this.state.stores} numItems={this.state.items.length}/>;
-
-    const method = (this.state.method === "location") ? <h5>Calculated from tracked location</h5> : <h5>Calculated from Zip Code: {this.state.zipCode}</h5>
   
     const shareButton = (this.state.originalStores.length === 0) ? null :
-        (<Button id="back-button" variant="contained" color="primary" onClick={this.onShare}>
+        (<Button id="back-button" variant="contained" onClick={this.onShare}>
            Share Stores <MailOutlineIcon color='white' />
```

Comment:
Try adding some space between the mail icon and the text. Also, we can change it from "Share stores" to "Share results"?